lib/client.js

```diff
@@ -0,0 +1,98 @@
+"use strict";
+
+const Promise = require("bluebird");
```

Comment:
I'm already looking at this. I am fixing up the tests to not depend on bluebird and will put in a conditional import